modules/installation-network-user-infra.adoc

```diff
@@ -270,6 +270,11 @@ ifdef::vsphere[]
 :!vsphere:
 endif::[]
 
+[discrete]
+== NTP configuration
+
+An {product-title} cluster that is installed in a restricted network is configured to use a public NTP server by default. To avoid clock skew, reconfigure the cluster to use a private NTP server instead. See the documentation for _Configuring chrony time service_ for more information.
```

Comment:
Should this use a link to the configuration documentation?
```suggestion
An {product-title} cluster that is installed in a restricted network is configured to use a public NTP server by default. To avoid clock skew, reconfigure the cluster to use a private NTP server instead. See the documentation for [Configuring chrony time service] for more information.
```

Reply:
Ideally, but this is a limitation of our tooling - I added it to additional resources as a mitigation. If all the places where this appears are correct, I can go add the additional resource link to them as well.

Reply:
OK. Makes sense. I'm still learning all the nuances of how the docs work, so I did not know if it was an oversight or a limitation.